src/modules/patient/patient.service.ts

```diff
@@ -214,7 +214,11 @@ export class PatientService {
         patient.name = capitalizeAllCharacters(body.name);
         patient.lastname = capitalizeAllCharacters(body.lastname);
         patient.secondLastname = capitalizeAllCharacters(body.secondLastname);
-        patient.birthDay = new Date(body.birthDate);
+        const parts = body.birthDate.split('-');
+        const year = parseInt(parts[0], 10);
+        const month = parseInt(parts[1], 10) - 1;
+        const day = parseInt(parts[2], 10);
+        patient.birthDay = new Date(year, month, day);
```

Comment:
can u create a function with a try catch to parse this date? since we can throws and exception could be greate to have it into a method and then you can show an snackbar error if something is wrong

Reply:
something like getPatientBirthDate()

Reply:
@imadiaz I have already implemented it